charts/opentelemetry-collector/UPGRADING.md

```diff
@@ -8,6 +8,20 @@ Just upgrade and enjoy.
 
 We broke the selector labels in `0.110.0`, which causes `helm upgrades` to fail. Do not attempt to upgrade from `0.110.0` to either `0.110.1` or `0.110.2`. Go straight to `0.110.3` instead.
 
+## 0.99.0 to 0.100.0
+
+If `memory_limiter` is set to `null`, it will no longer be overridden with values derived from k8s resource limits.
+Instead, the processor will be omitted from the final configuration.
+
+To maintain the same behavior as the previous version, replace `memory_limiter: null` with the following:
+
+```yaml
+memory_limiter:
+  check_interval: 5s
+  limit_percentage: 80
+  spike_limit_percentage: 25
```

Comment:
Maybe add a little more context to these notes for ease of use.

```suggestion
config:
  processors:
    memory_limiter:
      check_interval: 5s
      limit_percentage: 80
      spike_limit_percentage: 25
```

Reply:
Done!